src/pdf/DriverOverview/DriverOverviewPdfButton.tsx

```diff
@@ -74,8 +74,11 @@ const getParcelsForDelivery = async (parcelIds: string[]): Promise<ParcelsForDel
         }
 
         let labelCount = 0;
-        if (parcel.events && parcel.events.length > 0 && parcel.events[0].event_data) {
-            labelCount = Number.parseInt(parcel.events[0].event_data);
+        if (parcel.events && parcel.events.length > 0) {
+            const lastEventDataValue = parcel.events.slice(-1)[0].event_data;
+            if (lastEventDataValue) {
+                labelCount = Number.parseInt(lastEventDataValue);
+            }
```

Comment:
I feel like this logic could be simplified to allow for labelCount to be a const
maybe something like:
```markdown
const lastEventDataValue = parcel.events && parcel.events.length > 0 ?
     parcel.events.slice(-1)[0].event_data : 
     undefined;
const labelCount = lastEventDataValue ? Number.parseInt(lastEventDataValue) : 0;
```